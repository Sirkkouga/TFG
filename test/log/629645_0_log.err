INFO - 06/04/24 15:51:23 - 0:00:00 - Read 1071 words from the vocabulary file.

INFO - 06/04/24 15:51:23 - 0:00:00 - 3005 words (1071 unique) in 44 sentences.
INFO - 06/04/24 15:51:23 - 0:00:00 - 49 unknown words (13 unique), covering 1.63% of the data.
INFO - 06/04/24 15:51:23 - 0:00:00 - pression: 22
INFO - 06/04/24 15:51:23 - 0:00:00 - Grou@@: 5
INFO - 06/04/24 15:51:23 - 0:00:00 - fl@@: 4
INFO - 06/04/24 15:51:23 - 0:00:00 - +@@: 3
INFO - 06/04/24 15:51:23 - 0:00:00 - Over@@: 3
INFO - 06/04/24 15:51:23 - 0:00:00 - istor@@: 2
INFO - 06/04/24 15:51:23 - 0:00:00 - Job@@: 2
INFO - 06/04/24 15:51:23 - 0:00:00 - tri@@: 2
INFO - 06/04/24 15:51:23 - 0:00:00 - G: 2
INFO - 06/04/24 15:51:23 - 0:00:00 - string: 1
INFO - 06/04/24 15:51:23 - 0:00:00 - u: 1
INFO - 06/04/24 15:51:23 - 0:00:00 - ames: 1
INFO - 06/04/24 15:51:23 - 0:00:00 - U: 1
INFO - 06/04/24 15:51:23 - 0:00:00 - Job completed successfully
INFO - 06/04/24 15:51:23 - 0:00:00 - Exiting after successful completion
