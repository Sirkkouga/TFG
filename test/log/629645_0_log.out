submitit INFO (2024-06-04 15:51:20,718) - Starting with JobEnvironment(job_id=629645, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:20,719) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629645_submitted.pkl
Saving the data to /home/usuaris/veu/javier.abella/TFG/test/java.valid.cl.bpe.pth ...
submitit INFO (2024-06-04 15:51:23,905) - Job completed successfully
submitit INFO (2024-06-04 15:51:23,910) - Exiting after successful completion
